cachito/web/config.py

```diff
@@ -26,6 +26,11 @@ class Config(object):
     CACHITO_USER_REPRESENTATIVES: List[str] = []
     CACHITO_WORKER_USERNAMES: List[str] = []
 
+    # Temp dir used by the Prometheus Flask Exporter to coalesce the metrics from the threads
+    PROMETHEUS_METRICS_TEMP_DIR = (
+        os.environ.get("PROMETHEUS_METRICS_TEMP_DIR") or tempfile.gettempdir()
+    )
```

Comment:
Should this default to `/tmp/prometheus-flask-metrics` (or the platform-independent equivalent thereof) if the env. var is not defined?

Reply:
So...  in the process of addressing previous comments, I broke the multiprocess bit.  As a result, I lost track of the fact the environment variable is required.  Changes tested, pushed up.

Reply:
Wait, so why do we need both the `PROMETHEUS_MULTIPROC_DIR` environment variable and the `PROMETHEUS_METRICS_TEMP_DIR` config option? I was under the impression that the latter would be all that's needed.

Reply:
> Wait, so why do we need both the `PROMETHEUS_MULTIPROC_DIR` environment variable and the `PROMETHEUS_METRICS_TEMP_DIR` config option? I was under the impression that the latter would be all that's needed.

Fair question.   Due to the threaded nature of the application, the prometheus-flask-exporter package uses the temp directory for files to coalesce the metrics from the different threads.  I had this working when I initially created the code review.  In the process of shuffling some of the code around in response to comments (the function in metrics.py was in api_v1 originally),  the multithreaded bit broke (it still generated content on the /metrics endpoint, but not for every thread).  Its fixed.  The environmental variable is needed by the exporter module, no way around that.  The config option is to allow us a method of defining where.   

In the case of the container,  letting tempfile find the location is sufficient. 

In the case of the local dev environment, in particular with tox, being able to set it (and clean it up afterwards) is beneficial.